src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/UpdateSentenceDateService.kt

```diff
@@ -102,7 +103,12 @@ class UpdateSentenceDateService(
     )
 
     if (sentenceChanges.isMaterial) {
-      notifyComOfUpdate(updatedLicenceEntity, licenceEntity, licenceId, sentenceChanges)
+      val isNotApprovedForHdc = !hdcService.isApprovedForHdc(
+        updatedLicenceEntity.bookingId!!,
+        prisoner.sentenceDetail.homeDetentionCurfewEligibilityDate,
```

Comment:
previously we were not notifying the COM if they had a historic HDC case but were no longer eligible for HDC